lib/modules/manager/rust/extract.ts

```diff
@@ -0,0 +1,19 @@
+import { logger } from '../../../logger';
+import { GithubReleasesDatasource } from '../../datasource/github-releases';
+import type { PackageDependency, PackageFileContent } from '../types';
+
+export function extractPackageFile(content: string): PackageFileContent {
+  logger.trace('rust.extractPackageFile()');
+  const dep: PackageDependency = {
+    depName: 'rust',
+    packageName: 'rust-lang/rust',
+    currentValue: content.trim(),
```

Comment:
This will include the entire contents of the (toml) file as currentValue, which I think is incorrect